javascript-and-json/main.js

```diff
@@ -0,0 +1,28 @@
+var myArr = [
+  {
+    isbn: '121551-85858',
+    title: 'Harry Potter and the Order of Pheonix',
+    Author: 'J.K Rowling'
+  },
+  {
+    isbn: '121551-87787',
+    title: 'Harry Potter and Deathly Hallows',
+    Author: 'J.K Rowling'
+  },
+  {
+    isbn: '121551-585858',
+    title: 'Harry Potter and the Half-Blood Prince',
+    Author: 'J.K Rowling'
+  }
+];
+
+console.log('My array of books: ', myArr);
+console.log('Type of myArray ', typeof myArr);
+
+console.log('   JSON string of book array: ', JSON.stringify(myArr));
+console.log('Type of JSON string: ', typeof JSON.stringify(myArr));
+
+console.log('String in JSON FORM with single quotes ', '{ "name":"John", "Number ID": 37883}');
```

Comment:
The name of the property should just be `id`, not `Number ID`, that was just meant to tell you what the data type of the value should be 🤓